lib/chat_models/ollama_ai_fields.ex

```diff
@@ -0,0 +1,78 @@
+defmodule LangChain.ChatModels.OllamaAIFields do
+  defmacro fields do
+    quote do
+      @receive_timeout 60_000 * 5
```

Comment:
Is this really a 5 minute timeout? Why so long? Is it really slow to run these models locally? We might want to have the ability to override the timeout in that case.

Reply:
No clue why. The only information I found about time outs was here: https://github.com/jmorganca/ollama/pull/1257

Client should be able to override because it is a valid field in the struct